src/data/event.py

```diff
@@ -604,10 +609,14 @@ def get_unused_screen_uniq_id(
         """Returns the first unused screen uniq_id looking like base_uniq_id:
         base_uniq_id, or base_uniq_id-2, or base_uniq_id-n+1...
         screen_type is used when the given ID is empty to set an ID that corresponds to the screen type."""
-        assert base_uniq_id is not None or screen_type is not None
+        screen_uniq_id = base_uniq_id
+        if screen_uniq_id is None:
+            if screen_type is None:
+                raise ValueError('Either screen_type or base_uniq_id must be provided.')
```

Comment:
This can be:
```suggestion
        if screen_uniq_id is None and screen_type is None:
            raise ValueError('Either screen_type or base_uniq_id must be provided.')
```

Reply:
No it can't, otherwise screen_uniq_id = ... is no longer in an if statement